### DISCUSSION TITLE
logs

### DISCUSSION THREAD

**Discussion by surajit-zs** [gofr-dev/gofr#432] [Q&A] [answered]
<img width="592" alt="image" src="https://github.com/gofr-dev/gofr/assets/97510117/ff48834a-2888-4753-9adb-56dab97e0f9d">

here I have started my service in the default port with 8000 which is blocked ( another service is running in this port)
in that case, here I can see my service is still up. I don't know which port it is using.

**Comment by srijan-27** [comment, marked as answer]
@surajit-zs - Thanks for pointing out the issue. It was fixed as a part of #437
